python/poetry.lock

```diff
@@ -1,11 +1,12 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 2.1.1 and should not be changed by hand.
```

Comment:
This won't work. Manually done similar fix in https://github.com/rovio/rovio-ingest/pull/64/files.